### PR TITLE
add microphone

### PR DESCRIPTION
**Pull request by incognitojam**

### CODE REVIEW

File: log.capnp [hunk 1]
```diff
@@ -1961,6 +1961,11 @@ struct EncodeData {
 struct UserFlag {
 }
 
+struct Microphone {
+  noiseLevel @0 :Float32;
+  noiseLevelRaw @1 :Float32;
```
Comment (adeebshihadeh) [inline log.capnp:1966]: would add "ambient" prefix and "db" suffix

Comment (adeebshihadeh) [reply log.capnp:1966]: also add a comment saying the other one is filtered

**Review by adeebshihadeh — APPROVED** [review]
Feel free to merge after addressing comments.

File: log.capnp (hunk 1 above)
Comment (incognitojam) [reply log.capnp:1966]: I'm not sure it actually is dB

**Pull request merged**